src/lib/components/Widget.tsx

```diff
@@ -23,14 +23,15 @@ const WidgetWrapper = styled.div<{ width?: number | string }>`
   -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
   background-color: ${({ theme }) => theme.container};
   border-radius: ${({ theme }) => theme.borderRadius}em;
+  box-sizing: border-box;
   color: ${({ theme }) => theme.primary};
   display: flex;
   flex-direction: column;
   font-feature-settings: 'ss01' on, 'ss02' on, 'cv01' on, 'cv03' on;
   font-size: 16px;
   font-smooth: always;
   font-variant: none;
-  height: 348px;
+  height: 356px;
```

Comment:
is there any way to test this within an integration? i remember changing this didnt expose errors in cosmos but it did when used within other apps 

it affected the bottom amount of padding

Reply:
I manually tested this with widgets-demo to come up with this number.